eth_p2p/rlpx.nim

```diff
@@ -287,7 +287,10 @@ proc sendMsg*(peer: Peer, data: Bytes) {.async.} =
     discard await peer.transport.write(cipherText)
   except:
     await peer.disconnect(TcpError)
-    raise
+    # this is usually a "(32) Broken pipe":
```

Comment:
The intent of the previous code was to log the exception and re-raise it. This is more appropriate, because the users of the `sendMsg` call should be notified that their send operation have failed (in practice `sendMsg` is called by any of the generated high-level message sending procs in the RLPx protocols).

Reply:
Reverted to raising it and catching it in 3 other places. Also added a FIXME note for retrying failed attempts.